ts/features/walletV3/payment/components/WalletPaymentTotalAmount.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Placeholder from "rn-placeholder";
+import { StyleSheet, View } from "react-native";
+import { Divider, H3, H6, IOStyles } from "@pagopa/io-app-design-system";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import I18n from "../../../../i18n";
+
+type TotalAmountSectionProps = {
+  totalAmount?: number;
+  loading?: boolean;
+};
+
+const styles = StyleSheet.create({
+  container: {
+    ...IOStyles.rowSpaceBetween,
+    ...IOStyles.alignCenter,
+    ...IOStyles.flex,
+    paddingVertical: 16
+  }
+});
+
+export const WalletPaymentTotalAmount = ({
+  totalAmount,
+  loading
+}: TotalAmountSectionProps) => (
+  <>
+    <View style={styles.container}>
+      <H6 color="grey-700">{I18n.t("payment.confirm.totalAmount")}</H6>
+      {loading && (
+        <View>
+          <Placeholder.Box width={72} height={34} animate="fade" radius={8} />
+        </View>
+      )}
+      {!loading && totalAmount && (
+        <H3>{formatNumberCentsToAmount(totalAmount, true, "right")}</H3>
+      )}
```

Comment:
```suggestion
      {loading ? (
        <View>
          <Placeholder.Box width={72} height={34} animate="fade" radius={8} />
        </View>)
      
      : totalAmount && (
        <H3>{formatNumberCentsToAmount(totalAmount, true, "right")}</H3>
      )}
```

it does come down to preference, so feel free to dismiss my comment in case you like your implementation more, but overall I think moving from 2 checks to one both reduces redundancy and increases readability; What do you think?

Reply:
also, you could avoid the `totalAmount` check by using a format function with default value.
Not sure which one though, since we have a bunch of them 😅

Reply:
I think that using the ternary operator check or the and conditions is the same, Regarding the readability, I'm not sure that using the ternary operator instead of `&&` conditions is better. Imagine having a really big if-else statement, scrolling down, and losing the logical flow, seeing a condition in "else" with the `:` symbol, doesn't give you as much context and you would potentially find yourself forgetting the base condition, and going up in the code. Instead always inserting the condition might be more readable in cases like this as you've already indicated it's more of a personal preference.

About the `totalAmount`, we don't have a format function that accepts a nullable amount and I think is useless at the time declaring another formatter util since we already have a tons of that

Reply:
no sweat for the ternary, it's all good.
regarding the `totalAmount` thing though, we do have one! it's in `idpay/common/utils/strings.ts`, line 23 : `formatNumberCurrencyCentsOrDefault`.
I'm not really sure if the value is returned in cents from BE though, if not there is a "whole value" version in the same file 👍

Reply:
They are a lot! Used it in 7156aea62df0cd0940d1abf681c181ff18993281 commit